package.json

```diff
@@ -11,7 +11,24 @@
     "watermark": "./scripts/watermark.sh",
     "dev": "node server.js"
   },
+  "lint-staged": {
+    "*.css": [
+      "stylelint --fix",
```

Comment:
I'm curious -- where is the code that limits linting to _only_ files that have been staged? Is it something that husky already knows how to do?